spec/models/miq_task_spec.rb

```diff
@@ -1,181 +1,179 @@
 describe MiqTask do
   context "when I add an MiqTask" do
-    before(:each) do
-      @miq_task = FactoryGirl.create(:miq_task_plain)
-    end
+    let (:miq_task) { FactoryGirl.create(:miq_task_plain) }
```

Comment:
There shouldn't be a space between `let` and `(`, removing it will fix the rubocop comment.

Reply:
Fixed, thank you for letting me know.